code/modules/mob/living/simple_animal/abnormality/zayin/bottle.dm

```diff
@@ -176,22 +176,21 @@
 	user.AdjustSleeping(10 SECONDS)
 	if(user.stat == DEAD)
 		animate(user, alpha = 0, time = 2 SECONDS)
-		to_chat(user.client, span_userdanger("You have died from the bottle, and your body is now a part of the endless sea."))
```

Comment:
Why not replace it with anything if removing it?

Reply:
i just feel like its more consistent to have the user die without explanation, as happens with other abnormalities at the current moment.
When it was made i was more-so pivoting towards "let's tell them that they didn't have enough HP/Fort/Temp, and died due to that" but the message effectivelly served to do no explanation nor give flavor as i worded it wrongly.
If ya want i could still add the messages for all the different death reasons from bottle of tears tho